modules/ROOT/pages/email/email-gmail.adoc

```diff
@@ -91,3 +91,18 @@ Required parameters:
     </email:smtps-connection>
 </email:smtp-config>
 ----
+
+Additionally, if you experience SMTPS connection issues, check the following recommendations:
+
+If you have two-factor authentication set up:
+
+. There is no need for you to turn `Less Secure Apps` on
+. Generate an app specific password and use that one instead of your normal password. See https://support.google.com/accounts/answer/185833[Sign in using App Password] for details
+
+If you do not have two-factor authentication set up:
+
+. Turn `Less Secure Apps` on for your account
+. Use your normal password. If that does not work, go to https://accounts.google.com/DisplayUnlockCaptcha[Allow access to your Google account] site and:
+.. Enter your username and password
+.. Enter the letters on the screen
```

Comment:
Will the audience know what "the letters on the screen" refers to? What are these letters...aren't there a lot of letters on the screen?
```suggestion
.. Enter the letters on the screen.
```